rclrs/src/clock.rs

```diff
@@ -45,19 +46,37 @@ impl Clock {
     }
 
     /// Creates a new Clock with `ClockType::SteadyTime`
+    /// to update it
+    ///
+    /// # Returns
+    ///
+    /// A tuple of Clock and ClockSource
     pub fn steady() -> Self {
         Self::make(ClockType::SteadyTime)
     }
 
     /// Creates a new Clock with `ClockType::RosTime` and a matching `ClockSource` that can be used
     /// to update it
+    ///
+    /// # Returns
+    ///
+    /// A tuple of Clock and ClockSource
     pub fn with_source() -> (Self, ClockSource) {
         let clock = Self::make(ClockType::RosTime);
         let clock_source = ClockSource::new(clock.rcl_clock.clone());
         (clock, clock_source)
     }
 
     /// Creates a new clock of the given `ClockType`.
+    ///
+    /// # Arguments
```

Comment:
Same thing as the `# Returns` section. Rustdoc already does a good job showcasing the function signature with links to the referenced types. No need to add this manually, its a tad noisy